build/icon-builder/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM php:7
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        npm git sudo \
+        libnss3 libxss1 libasound2 libx11-xcb1 libxcb-dri3-0 libxcomposite1 \
+        libxcursor1 libxdamage1 libxi6 libxtst6 libatk1.0-0 libatk-bridge2.0-0 \
+        libcups2 libxrandr2 libdrm2 libgbm1 libpangocairo-1.0-0 libgtk-3-0 && \
+    apt-get clean && \
+    npm install -g svgexport
+
+# Install composer
+RUN curl https://raw.githubusercontent.com/composer/getcomposer.org/main/web/installer -qL | php -- --quiet && \
```

```suggestion
RUN curl -qsSfL https://raw.githubusercontent.com/composer/getcomposer.org/main/web/installer | php -- --quiet && \
```
- Not sure whether in a pipe this is relevant, but added `-s` to suppress possibly unintended STDOUT output.
- Added `-S` to keep showing errors, otherwise suppressed by `-s`, to help debugging failures in the workflow.
- Added `-f` to assure that webserver error response documents are sent to STDERR and not to STDOUT where they would be unintentionally processed through he pipe:
    ```sh
    curl https://raw.githubusercontent.com/composer/getcomposer.org/main/web/nonexistent 2> /dev/null | cat
    404: Not Found
    ```

---

Good catch!